SimplenoteUITests/Generic.swift

```diff
@@ -65,9 +71,14 @@ class Table {
     }
 
     class func trashCell(noteName: String) {
-        Table.getCell(label: noteName).swipeLeft()
-        sleep(1)
-        Table.getCell(label: noteName).buttons[UID.Button.itemTrash].tap()
+        // `Trash Note` and `Delete note forever` buttons have different labels
+        // since #1191. To use the correct label, we need to know where we are.
```

Comment:
It's cool how GitHub adds a link to the PR here 😍 

But, nitpick, the comment may be read from somewhere other than GitHub. In such case, a combination of commit and GitHub PR link might be more robust.

```suggestion
        // since 07fcccf1039495768ecdf9909d3dbd1b255936cd
        // (https://github.com/Automattic/simplenote-ios/pull/1191).
        // To use the correct label, we need to know where we are.
```

Reply:
As a side note, this scenario would be (will be) simpler in a page object world. Both the note editor and the trash screens would have their own object implementing a the method like this one, but without any conditional logic.

Reply:
✅  🙇‍♂️